azavea/adb-mongolia-nua
# Add language switcher button and toggle language on click

Adds a few translation keys to prove the functionality works
as intended, and to allow us to include the files in a
deployment.

TODO: Add translate bindings/keys everywhere in the app
TODO: Switch map config section/vis titles to contain
      translation keys instead of text
### To Test

`npm run serve` then navigate to homepage. Click switcher button in top left, the text 'MONGOLIA: GEOGRAPHIC DATA VIEWER' should change, along with the clicked button's text.

![adb-lang-en](https://cloud.githubusercontent.com/assets/1818302/17403860/7f7f3d3c-5a27-11e6-9e3a-341c2b361cbc.png)
![adb-lang-mn](https://cloud.githubusercontent.com/assets/1818302/17403861/7f892f68-5a27-11e6-9d65-fc8275dc233d.png)
